app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -238,3 +241,12 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       .result
   }
 }
+
+class ScenesToProjectesTableQuery[M, U, C[_]](scenesToProjects: ScenesToProjects.TableQuery) extends LazyLogging {
+
+  def sortDefault: ScenesToProjects.TableQuery = {
+    scenesToProjects
+      .sortBy(_.sceneOrder.asc.nullsLast)
+      .sortBy(_.sceneId.asc)
```

Comment:
This will always sort by `sceneId.asc`. I think you meant to do `.sortBy(s => (s.sceneOrder.asc.nullsLast, s.sceneId.asc))`

Reply:
see http://slick.lightbend.com/doc/3.0.0/sql-to-slick.html#order-by

Reply:
Oh man, good catch!

Reply:
Fixed in https://github.com/raster-foundry/raster-foundry/pull/2939/commits/6e3a4fdd7740b7e9d2e8b617ea07e39899fa6883